Tests/PackageLoading/ConventionTests.swift

```diff
@@ -727,40 +726,16 @@ final class PackageBuilderTester {
             if let root = root {
                 XCTAssertEqual(module.sources.root, AbsolutePath(root), file: file, line: line)
             }
-            var sources = self.sources
-
-            for path in paths.lazy.map(RelativePath.init) {
-                let contains = sources.contains(path)
-                XCTAssert(contains, "\(path) not found in module \(module.name)", file: file, line: line)
-                if contains {
-                    sources.remove(path)
-                }
-            }
-
-            guard sources.isEmpty else {
-                return XCTFail("Unchecked sources in package \(self): \(sources)", file: file, line: line)
-            }
+            let sources = Set(self.module.sources.relativePaths.map{$0.asString})
+            XCTAssertEqual(sources, Set(paths), file: file, line: line)
```

Comment:
It would be good to keep a more verbose failure message like `"unexpected source files"` or some such, just so CI logs are more readable.